2003/kangraemin/app/src/main/java/com/example/kangraemin/model/AuthRepository.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.kangraemin.model
+
+import android.content.Context
+import com.example.kangraemin.model.local.datadao.AuthImpl
+import com.example.kangraemin.model.local.datamodel.Auth
+import io.reactivex.BackpressureStrategy
+import io.reactivex.Completable
+import io.reactivex.Flowable
+import io.reactivex.schedulers.Schedulers
+
+class AuthRepository {
+
+    fun getAuth(context: Context): Flowable<Auth> {
+        return Flowable
+            .create({ emitter ->
+                val db = AppDatabase.getInstance(context = context)
+                AuthImpl(db = db)
+                    .getAuth()
+                    .subscribeOn(Schedulers.io())
+                    .subscribe({
+                        emitter.onNext(it)
+                    }, { emitter.onError(it) })
```

Comment:
계속 반복되고 있는 문제인데 rx이벤트를 subscribe해준 뒤 dispose해주지 않으면 메모리릭이 발생해요.
그리고 이 경우에는 굳이 여기서 subscribe할 필요가 없어요.
Flowable.create로 랩핑하지 않아도 실행에 문제가 없어요.

Single을 Flowable로 바꾸고 싶다면 
```suggestion
                    .toFlowable(BackpressureStrategy.BUFFER)
```
와 같은 함수를 이용해야해요.

Reply:
수정하였습니다
2032c72